packages/core/android/src/main/java/io/sentry/react/RNSentrySDK.java

```diff
@@ -25,28 +25,51 @@ private RNSentrySDK() {
    *
    * @param context Android Context
    * @param configuration configuration options
+   * @param logger logger
    */
   public static void init(
       @NotNull final Context context,
-      @NotNull Sentry.OptionsConfiguration<SentryAndroidOptions> configuration) {
+      @NotNull Sentry.OptionsConfiguration<SentryAndroidOptions> configuration,
+      @NotNull ILogger logger) {
     try {
       JSONObject jsonObject =
           RNSentryJsonUtils.getOptionsFromConfigurationFile(context, CONFIGURATION_FILE, logger);
       ReadableMap rnOptions = RNSentryJsonUtils.jsonObjectToReadableMap(jsonObject);
+      if (rnOptions == null) {
+        logger.log(
+            SentryLevel.WARNING,
+            "Failed to load configuration file("
+                + CONFIGURATION_FILE
+                + "), starting with configuration callback.");
```

Comment:
I think this log is not necessary/is duplicate as the same information will be logged out in `getOptionsFromConfigurationFile`.